test/integration/prerender/test/index.test.js

```diff
@@ -1895,13 +1896,14 @@ describe('SSG Prerender', () => {
 
       nextConfig.replace('// target', `target: 'serverless',`)
       await fs.remove(join(appDir, '.next'))
-      await nextBuild(appDir)
+      await nextBuild(appDir, undefined, { cwd: appDir })
```

Comment:
Is there a difference between `undefined` for the second param here vs `[]` like line 1947 below?

Reply:
No difference, undefined just uses the default here https://github.com/vercel/next.js/blob/49c3e0a201d333d4b76863768860f4dc98a568b2/test/lib/next-test-utils.js#L260